php/libraries/NDB_Menu_Filter.class.inc

```diff
@@ -764,8 +764,9 @@ class NDB_Menu_Filter extends NDB_Menu
         }
 
         return array(
-                'Headers' => $headers,
-                'Data'    => $MappedData,
+                'Headers'      => $headers,
+                'Data'         => $MappedData,
+                'fieldOptions' => $this->fieldOptions,
```

Comment:
I'm still not crazy about this mixing of concerns for presentation and data, but this PR is still a step forward so I'm not going to hold it up over this..

Reply:
I think in the next round of refactoring we can move to request the options in a separate call.